packages/valory/contracts/gnosis_safe/contract.py

```diff
@@ -741,7 +740,7 @@ def get_safe_txs(
 
         ledger_api = cast(EthereumApi, ledger_api)
         factory_contract = cls.get_instance(ledger_api, contract_address)
-        entries = factory_contract.events.SafeMultiSigTransaction.createFilter(
+        entries = factory_contract.events.ExecutionSuccess.createFilter(
```

Comment:
Just for context:
[GnosisSafeL2.sol](https://github.com/safe-global/safe-contracts/blob/main/contracts/GnosisSafeL2.sol) has the `SafeMultiSigTransaction` event, [GnosisSafe.sol](https://github.com/safe-global/safe-contracts/blob/main/contracts/GnosisSafe.sol) doesn't have it.
`ExecutionSuccess` is shared by both.